src/main/resources/avro/readmethods.avdl

```diff
@@ -122,6 +116,12 @@ record SearchReadGroupSetsRequest {
   Only return read group sets with this name (case-sensitive, exact match).
   */
   union { null, string } name = null;
+  
+  /**
+  Specifying the id of a BioSample record will return only readgroups
+  with the given bioSampleId.
+  */
```

Comment:
This comment alone doesn't make it clear that the returned `readGroupSet`s will have their `readGroup`s filtered by this field.